### ISSUE TITLE
[Android]Map.Pins.Add doesn't work when page OnAppearing

### ISSUE THREAD

**Issue by amay077** [amay077/Xamarin.Forms.GoogleMaps#30]
### REPRO

```
protected override void OnAppearing()
{
    base.OnAppearing();

    var pinTokyo = new Pin()
    {
        Type = PinType.Place,
        Label = "Tokyo SKYTREE",
        Address = "Sumida-ku, Tokyo, Japan",
        Position = new Position(35.71d, 139.81d)
    };

    map.Pins.Add(pinTokyo);
    map.SelectedPin = pinTokyo;
    map.MoveToRegion(MapSpan.FromCenterAndRadius(pinTokyo.Position, Distance.FromMeters(5000)));
}
```

* [Android]Pin Tokyo doesn't display
* [iOS]Pin Tokyo is displayed but InfoWindow doesn't display

Is this

* Xamarin.Forms.GoogleMaps's bug?
* Xamarin.Forms.Maps's bug?
* Xamarin.GooglePlayservices.Maps's bug?
* "native" Google Maps Android API's bug?

I don't investigate about it yet.

**Comment by AntonAndr** [comment]
Have the same problem. Also Map.Pins.Add() doesn't work when Page and Application constructors

**Comment by amay077** [comment]
Yes, Map initialization should wait for [``OnMapReady``](https://developers.google.com/android/reference/com/google/android/gms/maps/OnMapReadyCallback).

**Comment by amay077** [comment]
I fixed this in 1.5.1-beta1.
